### ISSUE TITLE
Alternative outputcurrency

### ISSUE THREAD

**Issue by Reno007** [Mikadily/poloniexlendingbot#309]
If I add EUR as an alternative output currency (instead of BTC) for the html log it does not work (and the bot applies BTC instead). Does this work for any of you guys?

**Comment by laxdog** [comment]
Works fine for me : 

<img width="184" alt="screen shot 2017-04-28 at 4 29 29 pm" src="https://cloud.githubusercontent.com/assets/794502/25535712/fd90b86a-2c2f-11e7-934e-722f819d7b64.png">

Is there anything in the log to stdout?

**Comment by rnevet** [comment]
Same here, my bot is in EUR.
![image](https://cloud.githubusercontent.com/assets/1645822/25535820/b63eb85a-2c38-11e7-8eb3-69ae661fc5ca.png)

Try the following link: https://blockchain.info/tobtc?currency=EUR&value=1
This is the link the bot uses to get the EUR conversion rate, if it fails it will display BTC.

**Comment by Reno007** [comment]
![capture1](https://cloud.githubusercontent.com/assets/28149572/25536100/c5b1dc76-2c39-11e7-9785-b46155b57cdb.PNG)
![capture2](https://cloud.githubusercontent.com/assets/28149572/25536101/c5b71c4a-2c39-11e7-9224-fafb24e4542f.PNG)

**Comment by Reno007** [comment]
Hmm, this is what I get...

**Comment by laxdog** [comment]
Also, if you're running on pythonanywhere, it won't work. They block that address https://github.com/Mikadily/poloniexlendingbot/issues/262

**Comment by Reno007** [comment]
Ah that would be it....

**Comment by Reno007** [comment]
Thanks!

**Comment by rnevet** [comment]
We could actually move the entire currency conversion to HTML javascript side.
@yura-pakhuchiy  what do you think? exchange rates are anyway public protocols.

**Comment by laxdog** [comment]
Good idea, it could also be a drop down then.

**Comment by yura-pakhuchiy** [comment]
Yes, we can have BTC -> output currency conversion done in JS (allowing output currency switch in runtime).
However, it is better to keep altcoin -> BTC rate in bot it self. Requesting every altcoin rate from JS will greatly increase time between json retrieval and moment when data can be displayed to user. Maybe even leave BTC->USDT rate in bot (for fastest default option), but move BTC -> EUR/real USD/others to JS (if user wants alternative currencies).

**Comment by Reno007** [comment]
That would be great! Thanks guys, awesome project

**Comment by laxdog** [comment]
Can we close this and create another ticket for the work to come out of it if that's still happening @yura-pakhuchiy or @rnevet ?

**Comment by rnevet** [comment]
@AndreyIlinX can you elaborate on the solution in your branch?

**Comment by AndreyIlinX** [comment]
@rnevet 
Hello, in short, I use build_opener from  urllib2 instead of using urlopen from urllib, and it works

**Comment by rnevet** [comment]
Ok, why not use that as the first call?
